rosidl_generator_cpp/test/test_interfaces.cpp

```diff
@@ -22,34 +22,30 @@
 #include <algorithm>
 #include "test_array_generator.hpp"
 
+#include "rosidl_generator_cpp/msg/array_primitives.hpp"
+#include "rosidl_generator_cpp/msg/array_nested.hpp"
+// TODO(jacobperron): Add test case for ArrayBoundedPrimitivesNested (need interface)
```

Comment:
I'd like to know others thoughts on adding this (and the other missing interface files) to `test_interface_files`. This would help maintain the same level of test coverage in this package, but introduce several more interface files. I guess more interface files isn't a big worry since other packages could ignore those they are not interested in testing.

Reply:
Adding them for now might be ok. We should probably consider to consolidate some of the messages based on what exactly each of them is aiming to cover.

Reply:
I think it's a good thing to cover all combinations of "nested type of nested type":

- ArrayBoundedSequenceNested
- ArrayUnboundedSequenceNested
- BoundedSequenceArrayNested
- BoundedSequenceUnboundedSequenceNested
- UnboundedSequenceArrayNested
- UnboundedSequenceBoundedSequenceNested

Which happen to be those that are missing here.